pandas/io/json/json.py

```diff
@@ -355,9 +355,9 @@ def read_json(
     path_or_buf : a valid JSON string or file-like, default: None
         The string could be a URL. Valid URL schemes include http, ftp, s3,
         gcs, and file. For file URLs, a host is expected. For instance, a local
-        file could be ``file://localhost/path/to/table.json``
+        file could be ``file://localhost/path/to/table.json``.
```

Comment:
Can you check if this is rendered correctly? I think we had problems before because something similar to this.

Reply:
Is it the dot after the `` `` ``` that you think might be a problem?

Reply:
Yes, may be it was with single backticks, or that is was a letter, but I think we had things like:
```
`DataFrame`s
```
That were causing problems. We'll probably capture now since they were also generating warnings if I'm not wrong, but better to render this docstring and see that it shows as expected.

Reply:
My changes introduce no new errors.

The changes, including new ones since my last commit, reduce the number of errors to two (including examples failing, which I address in the next paragraph). These errors were pre-existing.

As for the examples failing, I mentioned this in Gitter two days or so ago and was told it's probably OK. The rub is that the expected output is split over multiple lines but the actual output is on one line. I researched it for a good while but none of the remedies I tried worked. If y'all know offhand how to fix it, I will gladly do it. Here is one of the errors:
```
Line 158, in pandas.read_json
Failed example:
    df.to_json(orient='split')
Expected:
    '{"columns":["col 1","col 2"],
      "index":["row 1","row 2"],
      "data":[["a","b"],["c","d"]]}'
Got:
    '{"columns":["col 1","col 2"],"index":["row 1","row 2"],"data":[["a","b"],["c","d"]]}'
```

As to the period after the two backticks, I'm not getting any errors as to that. Here is a render of the change on my end:
<img width="583" alt="Screen Shot 2019-07-08 at 3 49 19 PM" src="https://user-images.githubusercontent.com/30704827/60841708-191de000-a198-11e9-81f5-6454da2db725.png">